deploy.sh

```diff
@@ -23,7 +23,7 @@ fi
 
 msg "Removing the old website"
 pushd public
-git rm -rf *
+git rm -rf public/*
```

Comment:
Wait, we don't want to delete the whole folder because we need to preserve the `.git/` directory inside it. We should leave this script as-is ;)